now.json

```diff
@@ -23,6 +23,41 @@
         "Location": "form-elements/input/"
       }
     },
+    {
+      "src": "form-elements/input/",
+      "status": 301,
+      "headers": {
+        "Location": "components/input/"
+      }
+    },
+    {
+      "src": "form-elements/checkbox/",
+      "status": 301,
+      "headers": {
+        "Location": "components/checkbox/"
+      }
+    },
```

Comment:
It doesn't look like these forwards are working. https://paste-git-chore-break-out-form.twilio-dsys.vercel.app/form-elements/checkbox/

Reply:
Might be missing slashes, I'll push up something and see if it helps